addons/point_of_sale/tests/common.py

```diff
@@ -352,17 +352,30 @@ def _create_taxes(cls):
         tax10: 10%, included in product price
         tax21: 21%, included in product price
         """
-        def create_tag(name):
+        def create_tag(name, negate=False):
             return cls.env['account.account.tag'].create({
                 'name': name,
                 'applicability': 'taxes',
-                'country_id': cls.env.company.country_id.id
+                'country_id': cls.env.company.country_id.id,
+                'tax_negate': negate,
+                'tax_report_line_ids': [
+                    (0, 0, {
+                        'name': name,
+                        'tag_name': name,
+                        'report_id': cls.tax_report.id,
+                        'sequence': 10,
+                    })
+                ]
             })
+        cls.tax_report = cls.env['account.tax.report'].create({
+            'name': "Tax report",
+            'country_id': cls.company_data['company'].country_id.id,
+        })
 
         cls.tax_tag_invoice_base = create_tag('Invoice Base tag')
         cls.tax_tag_invoice_tax = create_tag('Invoice Tax tag')
-        cls.tax_tag_refund_base = create_tag('Refund Base tag')
-        cls.tax_tag_refund_tax = create_tag('Refund Tax tag')
+        cls.tax_tag_refund_base = create_tag('Refund Base tag', True)
+        cls.tax_tag_refund_tax = create_tag('Refund Tax tag', True)
```

Comment:
Isn't it overkill to test the `tax_audit_string` in the PoS ?  You can probably just make similar test just by creating a misc move.

Aside of that, if you really want to test it, you will need to get the tag created from the report line and not to create the report line from the tag. (Because the report.line creates 2 tags: a positive and a negative by itself and that's how we are really using those)

Reply:
@Levizar I did it because some changes in account were done specifically for POS (see the diff where I remove the hook https://github.com/odoo/odoo/pull/88859/files#diff-1e3bd6be3bfb83a37ec9fb800ce8b1c95afe0be90ff792874ae7299c320a2f6eL3878).

As it's also overridden in POS code, I wanted to show that nothing has impacted (unless the wanted behavior)